src/jit/compiler.h

```diff
@@ -5606,6 +5606,10 @@ protected :
                             O2K_SUBRANGE };
     struct AssertionDsc
     {
+        AssertionDsc()
+        {
+          memset(this, 0, sizeof(AssertionDsc));
```

Comment:
Can't we initialize the individual data members? Using `memset` like this isn't exactly C++ish and some compilers may have trouble optimizing such code.